mission/functions/systems/actions/fn_action_arrest_player.sqf

```diff
@@ -19,8 +19,8 @@
 	format ["<t color='#B700FF'>%1</t>", localize 'STR_vn_mf_arrest_player'],							// Title of the action
 	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_secure_ca.paa",	// Idle icon shown on screen
 	"\a3\ui_f\data\IGUI\Cfg\holdactions\holdAction_secure_ca.paa",	// Progress icon shown on screen
-	"(player getVariable ['vn_mf_db_player_group', 'MikeForce'] in ['MilitaryPolice', 'DacCong']) && player distance cursorTarget <= 5 && primaryWeapon player != "" &&{ vehicle player isEqualTo player && {cursorTarget isKindOf 'Man' && {alive cursorTarget}}}",	// Condition for the action to be shown
-	"player distance cursorTarget < 7",						// Condition for the action to progress
+	player getVariable ["isArrested", "false"] == "false" && "(player getVariable ['vn_mf_db_player_group', 'MikeForce'] in ['MilitaryPolice', 'DacCong']) && player distance cursorTarget <= 3 &&{ vehicle player isEqualTo player && {cursorTarget isKindOf 'Man' && {alive cursorTarget}}}",	// Condition for the action to be shown
```

Comment:
What are you trying to do with the `player getVariable ["isArrested", "false"] == "false" && ` part of this?